src/graphql/queries/clientFetcher.ts

```diff
@@ -0,0 +1,36 @@
+import { logger } from '../../utils/logger';
+
```

Comment:
Denne funksjonen var egentlig generert av graphql-codegen, men for å kunne tilpasse feilmåndteringen så måtte jeg dra den ut.